docs/guides/deployment.md

```diff
@@ -1,2 +1,158 @@
 # Deployment
+This section contains guides and advices related to Raccoon deployment.
+## Standalone
+This section contains standalone deployment options for Raccoon server. You can use this deployment method on your local machine to get started with Raccoon.
+### Local Machine
```

Comment:
Don't need the `Local Machine` section since it's already part of the main read me.